openmdao/components/param_comp.py

```diff
@@ -6,7 +6,21 @@
 from openmdao.core.component import Component
 
 class ParamComp(Component):
-    """A Component that provides an output to connect to a parameter."""
+    """
+    A Component that provides an output to connect to a parameter.
+
+    Options
+	-------
+    force_fd :  bool(False)
+        Set to True to finite difference this system.
+    form :  str(forward)
```

Comment:
oops.  didn't re-run that docstring since the class is going away, but forgot it will be around for a little while.  will fix in 2 sec.